docs/scaling/secrets.md

```diff
@@ -17,7 +18,7 @@ Also, take a look at [the API docs for `@secrets`](/api/step-decorators/secrets)
 
 :::info
 
-Currently, `@secrets` supports only AWS Secrets Manager. Contact us on
+Currently, `@secrets` supports AWS Secrets Manager and Azure Key Vault. Contact us on
```

Comment:
not in the scope of this PR but GCP is also supported, so could also be updated here

Reply:
yes and no. GCP support is coming soon - but there is a GCP credentials PR that needs to be upstreamed before the docs can say so. @oavdeev is working on upstreaming that.